contracts/test/MockRollup.t.sol

```diff
@@ -95,10 +95,17 @@ contract MockRollupTest is BLSMockAVSDeployer {
         //get commitment with illegal value
         (IEigenDAServiceManager.BlobHeader memory blobHeader, EigenDABlobUtils.BlobVerificationProof memory blobVerificationProof) = _getCommitment(pseudoRandomNumber);
 
+        IEigenDAServiceManager.QuorumBlobParam[] memory quorumBlobParamsCopy = new IEigenDAServiceManager.QuorumBlobParam[](2);
```

Comment:
better not to hardcode it, i.e. create the array with `blobHeader.quorumBlobParams.length` instead of 2